core/controllers/suggestion.py

```diff
@@ -763,20 +763,26 @@ def get(self, target_type: str, suggestion_type: str) -> None:
         exploration_id = self.normalized_request.get('exploration_id')
         exp_ids = [exploration_id] if exploration_id else []
         user_settings = user_services.get_user_settings(self.user_id)
+        # User_settings.preferred_translation_language_code is the language
+        # selected by used in language filter of contributor dashboard.
```

Comment:
Please proofread your comments (and, in general, the PR) before requesting a follow-up review. This comment doesn't scan properly.

Reply:
Sorry for the typo, Corrected now, thanks.